distribution/changelog.txt

```diff
@@ -48,6 +48,7 @@
 - Fix: [#19379] "No platforms" station style shows platforms on the Junior Roller Coaster.
 - Fix: [#19380] Startup crash when no sequences are installed and random sequences are enabled.
 - Fix: [#19391] String corruption caused by an improper buffer handling in GfxWrapString.
+- Fix: [#19445] Text not cleared using Ctrl+Backspace.
```

Comment:
Hasn't this bug only just been added. I.e. no changelog required.

Reply:
You are right.